### DISCUSSION TITLE
x-bind with attribute with underscore not working

### DISCUSSION THREAD

**Discussion by sanderjn** [alpinejs/alpine#3546] [7. Bugs (archived)]
I'm looking to dynamically give an `<img>` element a `data-large_image=""` attribute. I need this specific attribute so I can feed the images to the WooCommerce lightbox.

I try to do this as follows:
`:data-large_image="variable"`

Sadly, this doesn't work, as it renders in my HTML as: `data-large`.

Example in this pen:
https://codepen.io/younginternet/pen/BaqWgJX

**Comment by sanderjn** [comment]
I'm seeing this is a duplicate of https://github.com/alpinejs/alpine/discussions/3258 and https://github.com/alpinejs/alpine/discussions/3242 (last one has the answer)

**Reply by ekwoka** [reply]
I'll look into why it doesn't "just work" already and see if it makes sense to patch.
